configs/default/SJET-AOCODARCH7DUAL.config

```diff
@@ -1,5 +1,12 @@
 # Betaflight / STM32H743 (SH74) 4.3.0 Feb  1 2022 / 20:15:46 (3267f0417) MSP API: 1.44
 
+#define USE_ACCGYRO_BMI270
+#define USE_BARO_DPS310
+#define USE_BARO_DPS310
```

Comment:
duplicate

Reply:
Please provide link to board information for verification

Reply:
Yes, I noticed but thanks.